tests/api/ils/test_resolvers.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test Items resolvers."""
+
+from invenio_circulation.pidstore.pids import CIRCULATION_LOAN_PID_TYPE
+
+from invenio_app_ils.acquisition.api import ORDER_PID_TYPE
+from invenio_app_ils.document_requests.api import DOCUMENT_REQUEST_PID_TYPE
+from invenio_app_ils.documents.api import DOCUMENT_PID_TYPE, Document
+from invenio_app_ils.documents.indexer import DocumentIndexer
+from invenio_app_ils.eitems.api import EITEM_PID_TYPE, EItem
+from invenio_app_ils.eitems.indexer import EItemIndexer
+from invenio_app_ils.ill.api import BORROWING_REQUEST_PID_TYPE
+from invenio_app_ils.internal_locations.indexer import InternalLocationIndexer
+from invenio_app_ils.items.api import ITEM_PID_TYPE, Item
+from invenio_app_ils.items.indexer import ItemIndexer
+from invenio_app_ils.locations.api import LOCATION_PID_TYPE, Location
+from invenio_app_ils.locations.indexer import LocationIndexer
+from invenio_app_ils.patrons.api import PATRON_PID_TYPE, Patron
+from invenio_app_ils.patrons.indexer import PatronIndexer
+
+from invenio_app_ils.internal_locations.api import (  # isort:skip
+    INTERNAL_LOCATION_PID_TYPE,
+    InternalLocation,
+)
+
+
+def test_jsonresolvers(testdata, mocker):
```

Comment:
added test for most of the resolvers (not all of them, we can add them in the future)